worldengine/draw.py

```diff
@@ -146,6 +198,9 @@ def set_pixel(self, x, y, color):
             color = (color[0], color[1], color[2], 255)
         self.pixels[x, y] = color
 
+    def get_pixel(self, x, y):
+        return self.pixels[x, y]
```

Comment:
You can technically achieve this by calling ```get_item()```, i.e. "pixel-thingy[x, y]" - I don't know if that is the nicest habit, but it certainly works. So maybe this method can be left out.